doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -6,117 +6,397 @@ nav: /:path/Programming-guide/Key-platform-features/Endpoint-profiles/
+The list of client-side endpoint profile schemas created by a tenant developer for the application is shown in the **Client-side EP profile** schemas window, which can 
+be opened from the application menu on the navigation panel as illustrated by the following screenshot.
 
+<img src="admin-ui/Client-side endpoint profile schema.png">
 
-## Server-side endpoint profile update ##
-Server-side endpoint profile is designed to be accessed and modified by various server-side applications through Kaa REST API integration layer. Server-side profile is not accessible from endpoint SDK or other client application logic.
+To export the client-side EP profile schema, click **Export** in the last column of desired schema row and select export method from drop-down. 
+See [CT schema export support]({{root_url}}Programming-guide/Key-platform-features/Common-Type-Library/#ct-schema-export-support) for available export methods details.
+As a tenant developer, you can create new client-side EP schemas for the application as follows:
+
+1. In the **Client-side EP profile schemas** window for the application, click **Add schema**.
+2. In the **Add profile schema** window enter the name of the schema.
+3. Then create a schema using one of the two options:
+    1. Using the existing CT by clicking **Select existing type** and selecting exiting CT version from FQN and version drop-downs.
+    
+    <img src="admin-ui/Create client-side endpoint profile schema 1.png">
+    
+    2. Create new CT by clicking **Create new type**. In this case you will be redirected to **Add new type** window. Here you can create a schema either by using 
+    the schema form or by uploading a schema in the [Avro](http://avro.apache.org/docs/current/spec.html) format from a file.
+    
+    <img src="admin-ui/Create client-side endpoint profile schema 2.png">
+    
+4. Click **Add** at the top of the window to save the schema.
+
+If you want to review the added Avro schema, open the corresponding **Client-side EP profile schema** window by clicking the schema in the **Client-side EP profile schemas** window.
+
+<img src="admin-ui/View client-side endpoint profile schema.png">
+
+### REST API for Client-side endpoint profile
+
+Use this link [Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs/#TODO) for getting more information.
+
+## Server-side endpoint profile ##
+
+The server-side endpoint profile is initially generated at the stage of a new endpoint registration. By default, server side profile record is auto-generated based on 
+the latest server-side profile schema of particular application. Both server-side endpoint profile schema and data can be updated at run time using 
+[Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs/#TODO) call. In this case, endpoint membership in the endpoint groups is re-evaluated and updated to match the new endpoint profile.
```

Comment:
Both server-side endpoint profile schema and data can be updated at run time using <link> ~~call~~.